Networking/NetworkingTests/Encoder/ProductImageAssetTypeCodableTests.swift

```diff
@@ -0,0 +1,86 @@
+import XCTest
+import Photos
+@testable import Networking
+
+final class ProductImageAssetTypeCodableTests: XCTestCase {
+
+    func test_uiImage_encodes_and_decodes_correctly() throws {
+        // Given
+        let assetType = ProductImageAssetType.uiImage(
```

Comment:
Not related to this PR: We should move the key string values `phAsset` and `uiImage` from `ProductImageAssetType` into a constant and reuse it while encoding and decoding. This will help avoid mistakingly editing these key values. 

The same comment applies for `ProductImageStatus` and `ProductOrVariationID` as well.